bigflow/build/dev.py

```diff
@@ -0,0 +1,98 @@
+"""Build-related developer tools and helpers.
+
+Intented to be used by "CLI" and in "development environement".
+"""
+
+import sys
+import logging
+import tempfile
+import pickle
+import typing
+import os
+import os.path
+
+from typing import Optional, List, Union
+from pathlib import Path
+
+import bigflow.commons as bfc
+import bigflow.build.reflect
+
+
+logger = logging.getLogger(__name__)
+
+# Used by 'dist.py' to dump `bigflow.setup(...)` arguments.
+DUMP_PARAMS_SETUPPY_CMDARG = "__bigflow_dump_params"
+
+
+def read_setuppy_args(path_to_setup: Union[Path, str, None] = None) -> dict:
+    """Loads `setup.py`, returns all parameters of `bigflow.build.setup()` function.
+
+    This function doesn't unpack 'embeeded sdist' archive when package is installed via pip.
+    You could use `bigflow.build.materialize_setuppy` for such purposes, altough it is not recommended"""
```

Comment:
You could use `bigflow.build.materialize_setuppy` for such purposes, **although** it is not recommended"""